src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
```

Comment:
Your approach is good but it's better to define default value on func argument

```suggestion
  constructor(name, health = 100) {
    this.name = name;
    this.health = health;
```